notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -1,1771 +1,1508 @@
 {
```

Comment:
<p>Change third sentence:</p><p> </p><p>"You can use the Explainable AI SDK to send explanation requests to your deployed model and visualize the explanations."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>